chucks-pick3-web/angular-app/src/app/app.module.ts

```diff
@@ -32,6 +32,7 @@ import {SignUpConfirmationComponent} from 'src/app/components/sign-up-confirmati
 import {MemberService} from 'src/app/services/member.service/member.service';
 import { UserNameValidator } from './directives/user-name-validator.directive';
 import { PasswordValidator } from './directives/password.directive';
+import { GenerateCustomerService } from './services/generate-customer.service/generate-customer.service';
```

Comment:
Rename GenerateCustomerService to MockCustomerService.

Reply:
Renamed to MockCustomerService.